include/deal.II/lac/petsc_vector_base.h

```diff
@@ -1190,7 +1190,7 @@ namespace PETScWrappers
         PetscErrorCode ierr = VecGetOwnershipRange (vector, &begin, &end);
         AssertThrow (ierr == 0, ExcPETScError(ierr));
 
-        Vec locally_stored_elements = PETSC_NULL;
+        Vec locally_stored_elements = nullptr;
```

Comment:
Formally, PETSC_NULL is deprecated and they say we should use NULL, so I believe nullptr would be okay here too.